### PR TITLE
add html content and css mobile-first

### PR DESCRIPTION
**Pull request by Lordkaito**
In this branch:

1) Added html and css for work and contact section.

2) Worked on mobile-first css

3) All images, shapes and backgrounds added

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -61,6 +60,304 @@ <h1 class="about">
...
+            HTML
+          </p>
+        </div>
+        <div class="css-frame d-flex">
+          <img src="./images/css-frame.png" alt="">
+          <p class="css-frame-text">
+            CSS
+          </p>
+        </div>
+      </div>
```
Comment (DanteAlonsoHT) [inline index.html]: - [x] The list of project technologies (e.g., "JavasScript", "HTML" and "CSS") are a list `ul` and each of them a `li`
- [x] Do what I typed above with the rest of your list of project technologies.

File: index.html [hunk 2]
```diff
@@ -61,6 +60,304 @@ <h1 class="about">
...
+    </div>
+  </section>
+  <section class="contact">
+    <div class="contact-container d-flex">
+      <div class="contact-text">
+        <h1 class="main-contact">
+          Contact me
+        </h1>
```
Comment (DanteAlonsoHT) [inline index.html]: - [x] [OPTIONAL] For SEO best practices, you only use the `h1` tag once per HTML file, please take the suggestion to code better than yesterday 😉

File: index.html [hunk 3]
```diff
@@ -61,6 +60,304 @@ <h1 class="about">
...
+          If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it
+        </p>
+      </div>
+        <div class="column d-flex">
+          <input type="text" class="form-control" id="contact" placeholder="Your Name">
+          <input type="email" class="form-control" id="email" placeholder="Your Email">
+          <textarea class="form-control" id="message" placeholder="Your Message"></textarea>
+          <div class="get-in-touch">
+            Get in Touch
+          </div>
```
Comment (DanteAlonsoHT) [inline index.html]: - [x] Instead `div` tag, you should use the `a` or `button` tag for the "Get in Touch" button, also I can not see the button, please try to fix it with CSS, maybe using the `z-index` attribute.
- [x] Try to fix the margin/padding for `input` tags in your contact form.

image

**Review by DanteAlonsoHT — CHANGES_REQUESTED** [review]
<h1> Status: Required Changes 🚫 </h1>

Hi @Lordkaito, 

Amazing job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

<hr>

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Lordkaito) [reply index.html]: Can you let me know wich screen width were you using, because padding and margin are looking good for me on my project. Thanks.

**Review by AshakaE — APPROVED** [review]
### Hi @Lordkaito  🙋‍♀️, 

# STATUS APPROVED ✔
Impressive work, keep it up 👍 
Your project is complete!  There is nothing else to say other than... it's time to merge it 🔗 🚢 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] 

You can also consider:
`320px` as the least breakpoint a mobile device can go. 💡 



Cheers 🥂  and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**